src/renderer/components/+custom-resources/crd-resource-details.tsx

```diff
@@ -15,11 +15,10 @@ import { KubeObjectMeta } from "../kube-object-meta";
 import { Input } from "../input";
 import type { AdditionalPrinterColumnsV1 } from "../../../common/k8s-api/endpoints/custom-resource-definition.api";
 import { CustomResourceDefinition } from "../../../common/k8s-api/endpoints/custom-resource-definition.api";
-import { convertKubectlJsonPathToNodeJsonPath } from "../../utils/jsonPath";
+import { safeJSONPathValue } from "../../utils/jsonPath";
 import type { KubeObjectMetadata, KubeObjectStatus } from "../../../common/k8s-api/kube-object";
 import { KubeObject } from "../../../common/k8s-api/kube-object";
 import logger from "../../../common/logger";
-import { JSONPath } from "@astronautlabs/jsonpath";
```

Comment:
Can this dependency be removed from package.json as well?

Reply:
No, because it is still being used https://github.com/lensapp/lens/pull/6530/files#diff-165d77f8a773439c161bd8259ce449213d2890c41cea52516c68b00f0a1a82f1L6-L7